tests/test_skills/test_apy_estimation_abci/conftest.py

```diff
@@ -481,6 +481,70 @@ def optimize_task_result() -> PoolToHyperParamsWithStatusType:
     return {"test1": ({"x": 2.0}, False), "test2": ({"x": 2.4}, True)}
 
 
+@pytest.fixture
+def real_case_optimize_task_input_data() -> Tuple[
+    Dict[str, pd.DataFrame], Dict[str, Union[int, float]]
+]:
+    """Return the data of a real study case."""
```

Comment:
This is a fixture containing the data that cause the 4 agents e2e test to fail on [this](https://github.com/valory-xyz/open-autonomy/runs/7581932296?check_suite_focus=true) CI's run.